pydy/tests/test_utils.py

```diff
@@ -18,3 +22,61 @@ def test_sympy_equal_to_or_newer_than():
     if parse_version(SETUPTOOLS_VERSION) >= parse_version('8.0'):
         with assert_raises(ValueError):
             sympy_equal_to_or_newer_than('0.7.7', '0.7.6-git')
+
+def test_mass():
+
+    mass = Mass()
+    assert mass.symbol == Symbol('m')
+    assert mass.value == 1
```

Comment:
Its my attempt to provide default values and user have the option to change those initial conditions. mass.value will be used in system.initial_conditions